.eslintrc.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
     },
   ],
   rules: {
+    "arrow-body-style": "off",
```

Comment:
Looks like this rule prevents the syntax needed in order to return from `useEffect` which is how to trigger an action on dismount.

Reply:
removed this in favor of the implicit return syntax